### ISSUE TITLE
Focus state needed on search examples

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#1060]
**Describe the bug**
The following pages show examples of search fields, that could use a focus state.

**To Reproduce**
http://localhost:4000/components/searchfield/example-header-large.html - should have a focus state
http://localhost:4000/components/toolbar/example-non-collapsible-searchfield.html - should have a focus state

**Expected behavior**
We should add the border change and dropshadow effect like other inputs.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
